Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessagesCountDecorationView.swift

```diff
@@ -5,8 +5,8 @@
 import StreamChat
 import UIKit
 
-/// The date separator view that groups messages from the same day.
-open class ChatThreadRepliesCountDecorationView: ChatMessageDecorationView, AppearanceProvider {
+/// The view used to show a separator when there are unread messages.
+open class ChatMessagesCountDecorationView: ChatMessageDecorationView, AppearanceProvider {
     /// The date in string format.
     open var content: String? {
```

Comment:
I know this comes from a different PR, but it would be nice if this was actually a `ChatMessage` instead of a String. It will make it hard for customers to customise this

Reply:
A ChatMessage? It should actually display the number of unread messages, not the message

Reply:
I do agree with the above, though

Reply:
I'm not sure I agree with the content being `ChatMessage`. This is a view that is being reused between Thread Replies Counter and Unread Messages. Currently the logic of what message to show is handled higher in the hierarchy from the responsible VC. Making the content `ChatMessage` it will require this view to take more responsibility and make it more complext (potentially it will need to be split into separate classes).

Regarding customers being able to customise it, we offer full control from the delegate (where this view is coming from)

Reply:
That being said, we should probably change the comment above as it's not a date anymore :P